From 7ffb51139cdf8139840ce4f8caa61cbb133a6c12 Mon Sep 17 00:00:00 2001
From: Yann Collette <ycollette.nospam@free.fr>
Date: Sun, 10 Sep 2017 08:46:43 +0200
Subject: [PATCH] fix versioninfo

---
 include/versioninfo.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/include/versioninfo.h b/include/versioninfo.h
index 0fd3152..9295a56 100644
--- a/include/versioninfo.h
+++ b/include/versioninfo.h
@@ -1,5 +1,5 @@
 #ifdef __GNUC__
-#define GCC_VERSION "GCC "__VERSION__
+#define GCC_VERSION "GCC " __VERSION__
 #else
 #define GCC_VERSION "unknown compiler"
 #endif
-- 
2.7.5

