diff -rupN jamin-0.95.0.old/configure.in jamin-0.95.0/configure.in
--- jamin-0.95.0.old/configure.in	2008-11-30 09:42:14.000000000 -0500
+++ jamin-0.95.0/configure.in	2010-02-10 01:52:27.000000000 -0500
@@ -109,7 +109,7 @@ if test "${libdir}" = '${exec_prefix}/li
   libdir='${exec_prefix}/'${libnn}
 fi
 
-JAMIN_LIBS="$FFTW_LIBS $XML_LIBS $JACK_LIBS $OSC_LIBS"
+JAMIN_LIBS="$FFTW_LIBS $XML_LIBS $JACK_LIBS $OSC_LIBS -ldl"
 AC_SUBST(JAMIN_LIBS)
 JAMIN_CFLAGS="-g $FFTW_CFLAGS $JACK_CFAGS $XML_CFLAGS $OSC_CFLAGS"
 AC_SUBST(JAMIN_CFLAGS)
