diff -rupN non-ntk-20130730.org/src/filename_list.cxx non-ntk-20130730/src/filename_list.cxx
--- non-ntk-20130730.org/src/filename_list.cxx	2013-08-28 11:43:04.000000000 -0400
+++ non-ntk-20130730/src/filename_list.cxx	2017-02-18 14:42:19.087232676 -0500
@@ -101,7 +101,7 @@ int fl_filename_list(const char *d, dire
   fl_utf8to_mb(d, dirlen, dirloc, dirlen + 1);
 #endif
 
-#ifndef HAVE_SCANDIR
+#if !defined(HAVE_SCANDIR) && !defined(HAVE_SCANDIR_POSIX)
   // This version is when we define our own scandir
   int n = fl_scandir(dirloc, list, 0, sort);
 #elif defined(HAVE_SCANDIR_POSIX) && !defined(__APPLE__)
