--- jacktrip-1.0.5/src/JackTrip.cpp~	2009-06-30 09:28:52.000000000 -0700
+++ jacktrip-1.0.5/src/JackTrip.cpp	2010-05-14 17:31:48.000000000 -0700
@@ -211,6 +211,8 @@
   setupDataProtocol();
   setupRingBuffers();
 
+  mJackAudio->startProcess();
+
   // Start the threads for the specific mode
   switch ( mJackTripMode )
     {
@@ -229,7 +231,6 @@
     }
   
   // Start Threads
-  mJackAudio->startProcess();
   for (int i = 0; i < mProcessPlugins.size(); ++i) {
     mJackAudio->appendProcessPlugin(mProcessPlugins[i]);
   }
