From: Miller Puckette <msp@ucsd.edu>
Date: Fri, 11 Sep 2020 12:32:31 -0700
Subject: fix for filenames with spaces as arguments to pd~ "start" message

---
 extra/pd~/pd~.c | 5 ++++-
 1 file changed, 4 insertions(+), 1 deletion(-)

diff --git a/extra/pd~/pd~.c b/extra/pd~/pd~.c
index 162db9e..2720258 100644
--- a/extra/pd~/pd~.c
+++ b/extra/pd~/pd~.c
@@ -620,7 +620,10 @@ gotone:
     for (i = 0; i < argc; i++)
     {
 #ifdef PD
-        atom_string(&argv[i], tmpbuf, MAXPDSTRING);
+        if (argv[i].a_type == A_SYMBOL)
+            snprintf(tmpbuf, MAXPDSTRING, "%s", argv[i].a_w.w_symbol->s_name);
+        else if (argv[i].a_type == A_FLOAT)
+            sprintf(tmpbuf,  "%f", (float)argv->a_w.w_float);
 #endif
 #ifdef MSP
             /* because Mac pathnames sometimes have an evil preceeding
