From 1577943aa5135f1bb8ce061b1fb917dba1d8d855 Mon Sep 17 00:00:00 2001
From: Yann Collette <ycollette.nospam@free.fr>
Date: Tue, 29 Dec 2020 11:11:32 +0100
Subject: [PATCH] add attributes

---
 vocoder.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/vocoder.c b/vocoder.c
index 38d5a1a..477dd44 100644
--- a/vocoder.c
+++ b/vocoder.c
@@ -277,7 +277,7 @@ LADSPA_Descriptor * g_psDescriptor = NULL;
 
 /* _init() is called automatically when the plugin library is first
    loaded. */
-void 
+static void __attribute__ ((constructor))
 _init() {
   char ** pcPortNames;
   LADSPA_PortDescriptor * piPortDescriptors;
@@ -352,7 +352,7 @@ _init() {
 /*****************************************************************************/
 
 /* _fini() is called automatically when the library is unloaded. */
-void 
+static void __attribute__ ((destructor))
 _fini() {
   long lIndex;
   if (g_psDescriptor) {
-- 
2.26.2

