From 0faa283cba2a9be86dd3a078e40a235732f0094c Mon Sep 17 00:00:00 2001
From: Yann Collette <yann.collette@artelys.com>
Date: Thu, 1 Oct 2020 14:48:32 +0200
Subject: [PATCH] add missing header

---
 src/ladspa_plugin/ladspa++.h | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/src/ladspa_plugin/ladspa++.h b/src/ladspa_plugin/ladspa++.h
index 190914e..648547d 100644
--- a/src/ladspa_plugin/ladspa++.h
+++ b/src/ladspa_plugin/ladspa++.h
@@ -21,6 +21,8 @@
 #include <tuple>
 #include <array>
 #include <cassert>
+#include <stdint.h>
+#include <stddef.h>
 
 #include "ladspa.h"
 
-- 
2.26.2

