From 9a595b258f8c7324d8e8b30122df85652fdb062e Mon Sep 17 00:00:00 2001
From: Yann Collette <ycollette.nospam@free.fr>
Date: Tue, 20 Oct 2020 13:55:47 +0200
Subject: [PATCH] fix flags override

---
 CMakeLists.txt | 15 ---------------
 1 file changed, 15 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 49ee84c..b9c47f0 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -117,21 +117,6 @@ ELSE()
     SET(H2CORE_LIBRARY_TYPE STATIC)
 ENDIF()
 
-IF(WANT_DEBUG)
-	SET(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -O0")
-ELSE()
-	SET(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -O2")
-ENDIF()
-
-IF (NOT CMAKE_CXX_COMPILER_ID MATCHES "Clang")
-    SET(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -fno-implement-inlines")
-ENDIF()
-
-SET(CMAKE_CXX_FLAGS_RELEASE "")
-
-SET(CMAKE_CXX_FLAGS_DEBUG "-g ")#-Winline")
-
-
 IF(APPLE)
 	#Compile for 32bit and 64bit intel macs
 	SET (CMAKE_OSX_ARCHITECTURES "x86_64;")
-- 
2.26.2

