From 1b47fb79ddf4e26a97ea36a3f54b95bf930dec6d Mon Sep 17 00:00:00 2001
From: Yann Collette <ycollette.nospam@free.fr>
Date: Mon, 5 Oct 2020 15:41:53 +0200
Subject: [PATCH] fix for lua 54

---
 backends/lua.c | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/backends/lua.c b/backends/lua.c
index 98ce369..e3e2278 100644
--- a/backends/lua.c
+++ b/backends/lua.c
@@ -144,7 +144,8 @@ static void lua_thread_resume(size_t current_thread){
 
 	//call thread main
 	DBGPF("Resuming thread %" PRIsize_t " on %s", current_thread, thread[current_thread].instance->name);
-	if(lua_resume(thread[current_thread].thread, NULL, 0) != LUA_YIELD){
+	int nResults = 0;
+	if(lua_resume(thread[current_thread].thread, NULL, 0, &nResults) != LUA_YIELD){
 		DBGPF("Thread %" PRIsize_t " on %s terminated", current_thread, thread[current_thread].instance->name);
 		thread[current_thread].timeout = 0;
 	}
-- 
2.28.0

