From 32de2b449b25eee91a3b3ab52a11f3d11215f7e7 Mon Sep 17 00:00:00 2001
From: Yann Collette <yann.collette@artelys.com>
Date: Fri, 1 May 2020 11:20:08 +0200
Subject: [PATCH] remove uninstall target

---
 CMakeLists.txt | 7 -------
 1 file changed, 7 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 5834442..c5ce787 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -128,13 +128,6 @@ CONFIGURE_FILE(
     "${CMAKE_SOURCE_DIR}/kmidimon.spec"
     IMMEDIATE @ONLY)
 
-CONFIGURE_FILE(
-    "${CMAKE_SOURCE_DIR}/cmake_admin/cmake_uninstall.cmake.in"
-    "${CMAKE_CURRENT_BINARY_DIR}/cmake_uninstall.cmake"
-    IMMEDIATE @ONLY)
-ADD_CUSTOM_TARGET(uninstall
-    "${CMAKE_COMMAND}" -P "${CMAKE_CURRENT_BINARY_DIR}/cmake_uninstall.cmake")
-
 ADD_CUSTOM_TARGET ( tarball
     COMMAND mkdir -p kmidimon-${VERSION}
     COMMAND cp -r cmake_admin kmidimon-${VERSION}
-- 
2.26.2

