From 27d1daae35e4e6a8222dcf0383e1f52a7c84d65f Mon Sep 17 00:00:00 2001
From: Yann Collette <ycollette.nospam@free.fr>
Date: Thu, 1 Apr 2021 22:19:27 +0200
Subject: [PATCH] use nullptr

---
 src/connectdlg.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/connectdlg.cpp b/src/connectdlg.cpp
index dea12a7..1ee3a46 100644
--- a/src/connectdlg.cpp
+++ b/src/connectdlg.cpp
@@ -47,7 +47,7 @@ ConnectDlg::ConnectDlg( QWidget *parent,
     QVBoxLayout* vbl2 = new QVBoxLayout(m_group);
     for (int i = 0; i < inputs.size(); ++i) {
         QCheckBox *chk = new QCheckBox(inputs[i], m_group);
-        chk->setChecked(subs.contains(inputs[i]) > 0);
+        chk->setChecked(subs.contains(inputs[i]) != nullptr);
         vbl2->addWidget(chk);
     }
     QLabel* lbl = new QLabel(i18n("<b>Output Connection:</b>"), w);
-- 
2.30.2

