Index: SyntopiaCore/GLEngine/Raytracer/VoxelStepper.cpp
===================================================================
--- SyntopiaCore/GLEngine/Raytracer/VoxelStepper.cpp	(révision 312)
+++ SyntopiaCore/GLEngine/Raytracer/VoxelStepper.cpp	(copie de travail)
@@ -122,7 +122,7 @@
 						currentT = p;
 
 						// We do not intersect grid.
-						if (!found) return false;
+						if (!found) return nullptr;
 				}
 
 				stepX = (dir.x() > 0) ? 1 : -1;
