From fa2be3284bad992c1c27e8ea586107775c171e15 Mon Sep 17 00:00:00 2001
From: Yann Collette <ycollette.nospam@free.fr>
Date: Sun, 13 May 2018 13:56:33 +0200
Subject: [PATCH] fix math include

---
 dsp/v4f_IIR2.h | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/dsp/v4f_IIR2.h b/dsp/v4f_IIR2.h
index a68ecf5..2f9d5a6 100644
--- a/dsp/v4f_IIR2.h
+++ b/dsp/v4f_IIR2.h
@@ -28,6 +28,9 @@
 #ifndef DSP_V4F_IIR2_H
 #define DSP_V4F_IIR2_H
 
+#include <math.h>
+#define pow10f(x) exp10f(x)
+
 #include "v4f.h"
 
 namespace DSP {
-- 
2.13.6

