From 45cc2e9e0132963d78272bb6597767a90b125181 Mon Sep 17 00:00:00 2001
From: Yann Collette <ycollette.nospam@free.fr>
Date: Tue, 27 Oct 2020 08:13:10 +0100
Subject: [PATCH] add missing include

---
 c++/widgets/pixmapdial.cpp | 1 +
 1 file changed, 1 insertion(+)

diff --git a/c++/widgets/pixmapdial.cpp b/c++/widgets/pixmapdial.cpp
index 37dcffd..672ce36 100644
--- a/c++/widgets/pixmapdial.cpp
+++ b/c++/widgets/pixmapdial.cpp
@@ -22,6 +22,7 @@
 #include <QtCore/QTimer>
 #include <QtGui/QPainter>
 #include <QtGui/QPaintEvent>
+#include <QPainterPath>
 
 PixmapDial::PixmapDial(QWidget* parent)
     : QDial(parent),
-- 
2.26.2

