From 448a6b6502279ffdab04461b7385d74a6a73bdba Mon Sep 17 00:00:00 2001
From: Yann Collette <yann.collette@artelys.com>
Date: Fri, 2 Oct 2020 15:39:06 +0200
Subject: [PATCH] add missing header

---
 objects/nxcurve.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/objects/nxcurve.h b/objects/nxcurve.h
index a4b493e..0a1623f 100644
--- a/objects/nxcurve.h
+++ b/objects/nxcurve.h
@@ -28,6 +28,7 @@
+#include <QPainterPath>
 #include <QScriptValue>
 #include <QVarLengthArray>
 #include <QBitmap>
 #include "geometry/qmuparser/muParser.h"
 #include "nxobject.h"
 #include "qmath.h"
-- 
2.26.2

