From af04a1a7a78b8455cafa3b203ef3d56a27007006 Mon Sep 17 00:00:00 2001
From: Yann Collette <ycollette.nospam@free.fr>
Date: Sun, 24 Jul 2016 13:39:02 +0200
Subject: [PATCH] fix ressources generation

---
 src/toolkits/embedresources.cpp | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/toolkits/embedresources.cpp b/src/toolkits/embedresources.cpp
index 673a8e9..cb90a12 100644
--- a/src/toolkits/embedresources.cpp
+++ b/src/toolkits/embedresources.cpp
@@ -25,14 +25,14 @@ void append_file(FILE *fout, const char *path, const char *name)
     FILE *f = fopen(path, "rb");
     unsigned char c;
     unsigned int i = 0;
-    fprintf(fout, "const char _%s[] = {\n", name);
+    fprintf(fout, "const unsigned char _%s[] = {\n", name);
     while (!feof(f))
     {
         fread(&c, 1, 1, f);
         fprintf(fout, "0x%02X%s%s", c, !feof(f) ? "," : "", ++i % 16 ? "" : "\n");
     }
     fprintf(fout, "};\n");
-    fprintf(fout, "const char *%s = _%s;\n", name, name);
+    fprintf(fout, "const unsigned char *%s = _%s;\n", name, name);
     fclose(f);
 }
 
-- 
2.7.4

