From 95ee8c859e66fef7d51538b700bcbe52d54b6fb7 Mon Sep 17 00:00:00 2001
From: Yann Collette <ycollette.nospam@free.fr>
Date: Sat, 15 May 2021 22:01:15 +0200
Subject: [PATCH] use nullptr

---
 lib/rdcart.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/lib/rdcart.cpp b/lib/rdcart.cpp
index 0a0fb09..dc1d302 100644
--- a/lib/rdcart.cpp
+++ b/lib/rdcart.cpp
@@ -362,7 +362,7 @@ void RDCart::updateSchedCodes(const QString &add_codes,const QString &remove_cod
   	QString wstr=q->value(0).toString();
   	wstr+="          ";
         wstr=wstr.left(11);
-  	if((sched_codes.contains(wstr)>0||add_codes.contains(wstr)>0)&&remove_codes.contains(wstr)==0) {
+  	if((sched_codes.contains(wstr)!=nullptr||add_codes.contains(wstr)!=nullptr)&&remove_codes.contains(wstr)==0) {
           save_codes.push_back(wstr.stripWhiteSpace());
   	}
   }
-- 
2.31.1

