From aec229e276628c57362baba60b5fdc2d0488613d Mon Sep 17 00:00:00 2001
From: Yann Collette <ycollette.nospam@free.fr>
Date: Sat, 15 May 2021 22:34:03 +0200
Subject: [PATCH] use nullptr 2

---
 lib/rdweb.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/lib/rdweb.cpp b/lib/rdweb.cpp
index 8a402ae..43d7c53 100644
--- a/lib/rdweb.cpp
+++ b/lib/rdweb.cpp
@@ -805,7 +805,7 @@ bool RDParsePost(std::map<QString,QString> *vars)
   // Get message parts
   //
   while((n=getline(&data,(size_t *)&n,f))>0) {
-    if(QString(data).stripWhiteSpace().contains(sep)>0) {  // End of part
+    if(QString(data).stripWhiteSpace().contains(sep)!=nullptr) {  // End of part
       if(fd>=0) {
 	ftruncate(fd,lseek(fd,0,SEEK_CUR)-2);  // Remove extraneous final CR/LF
 	::close(fd);
-- 
2.31.1

