From a59045cf2fa8d654f2d42ee6dc723d6269383671 Mon Sep 17 00:00:00 2001
From: Yann Collette <ycollette.nospam@free.fr>
Date: Sat, 15 May 2021 22:53:58 +0200
Subject: [PATCH] use nullptr 3

---
 importers/nexgen_filter.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/importers/nexgen_filter.cpp b/importers/nexgen_filter.cpp
index 7cbc066..b7e06b1 100644
--- a/importers/nexgen_filter.cpp
+++ b/importers/nexgen_filter.cpp
@@ -641,7 +641,7 @@ QDateTime MainObject::GetDateTime(const QString &str) const
 QString MainObject::SwapCase(const QString &str) const
 {
   QStringList parts=str.split(".");
-  if(parts[parts.size()-1].contains(QRegExp("*[a-z]*",true,true))>0) {
+  if(parts[parts.size()-1].contains(QRegExp("*[a-z]*",true,true))!=nullptr) {
     parts[parts.size()-1]=parts[parts.size()-1].upper();
   }
   else {
-- 
2.31.1

