unchanged:
--- a/FugioApp/contextwidgetprivate.cpp
+++ b/FugioApp/contextwidgetprivate.cpp
@@ -4,6 +4,7 @@
 #include <QFileDialog>
 #include <QStylePainter>
 #include <QStandardPaths>
+#include <QStyleOption>
 
 #include "app.h"
 #include "contextview.h"
diff -u b/plugins/OpenCV/grayscalenode.cpp b/plugins/OpenCV/grayscalenode.cpp
--- b/plugins/OpenCV/grayscalenode.cpp
+++ b/plugins/OpenCV/grayscalenode.cpp
@@ -62,26 +62,26 @@
 	switch( SrcImg.format() )
 	{
 		case fugio::ImageFormat::RGB8:
-			cv::cvtColor( MatSrc, pNode->mMatImg, CV_RGB2GRAY );
+			cv::cvtColor( MatSrc, pNode->mMatImg, cv::COLOR_RGB2GRAY );
 			break;
 
 		case fugio::ImageFormat::BGR8:
-			cv::cvtColor( MatSrc, pNode->mMatImg, CV_BGR2GRAY );
+		  cv::cvtColor( MatSrc, pNode->mMatImg, cv::COLOR_BGR2GRAY );
 			break;
 
 		case fugio::ImageFormat::RGBA8:
-			cv::cvtColor( MatSrc, pNode->mMatImg, CV_RGBA2GRAY );
+			cv::cvtColor( MatSrc, pNode->mMatImg, cv::COLOR_RGBA2GRAY );
 			break;
 
 		case fugio::ImageFormat::BGRA8:
-			cv::cvtColor( MatSrc, pNode->mMatImg, CV_BGRA2GRAY );
+			cv::cvtColor( MatSrc, pNode->mMatImg, cv::COLOR_BGRA2GRAY );
 			break;
 
 //		case fugio::ImageFormat::GRAY16:
 //			break;
 
 //		case fugio::ImageFormat::YUYV422:
-//			cv::cvtColor( MatSrc, mMatImg, CV_YUV2GRAY_420 );
+//			cv::cvtColor( MatSrc, mMatImg, cv::COLOR_YUV2GRAY_420 );
 //			break;
 
 		default:
unchanged:
--- a/plugins/OpenCV/imageconvertnode.cpp
+++ b/plugins/OpenCV/imageconvertnode.cpp
@@ -128,11 +128,11 @@ void ImageConvertNode::inputsUpdated( qint64 pTimeStamp )
 			switch( mDestinationFormat )
 			{
 				case fugio::ImageFormat::BGR8:
-					CvtCod = CV_HSV2BGR;
+					CvtCod = cv::COLOR_HSV2BGR;
 					break;
 
 				case fugio::ImageFormat::RGB8:
-					CvtCod = CV_HSV2RGB;
+					CvtCod = cv::COLOR_HSV2RGB;
 					break;
 
 				default:
@@ -144,11 +144,11 @@ void ImageConvertNode::inputsUpdated( qint64 pTimeStamp )
 			switch( mDestinationFormat )
 			{
 				case fugio::ImageFormat::GRAY8:
-					CvtCod = CV_RGB2GRAY;
+					CvtCod = cv::COLOR_RGB2GRAY;
 					break;
 
 				case fugio::ImageFormat::HSV8:
-					CvtCod = CV_RGB2HSV;
+					CvtCod = cv::COLOR_RGB2HSV;
 					break;
 
 				default:
@@ -160,7 +160,7 @@ void ImageConvertNode::inputsUpdated( qint64 pTimeStamp )
 			switch( mDestinationFormat )
 			{
 				case fugio::ImageFormat::RGB8:
-					CvtCod = CV_RGBA2RGB;
+					CvtCod = cv::COLOR_RGBA2RGB;
 					break;
 
 				default:
@@ -172,15 +172,15 @@ void ImageConvertNode::inputsUpdated( qint64 pTimeStamp )
 			switch( mDestinationFormat )
 			{
 				case fugio::ImageFormat::GRAY8:
-					CvtCod = CV_BGR2GRAY;
+					CvtCod = cv::COLOR_BGR2GRAY;
 					break;
 
 				case fugio::ImageFormat::RGB8:
-					CvtCod = CV_BGR2RGB;
+					CvtCod = cv::COLOR_BGR2RGB;
 					break;
 
 				case fugio::ImageFormat::HSV8:
-					CvtCod = CV_BGR2HSV;
+					CvtCod = cv::COLOR_BGR2HSV;
 					break;
 
 				default:
@@ -192,11 +192,11 @@ void ImageConvertNode::inputsUpdated( qint64 pTimeStamp )
 			switch( mDestinationFormat )
 			{
 				case fugio::ImageFormat::GRAY8:
-					CvtCod = CV_BGRA2GRAY;
+					CvtCod = cv::COLOR_BGRA2GRAY;
 					break;
 
 				case fugio::ImageFormat::RGB8:
-					CvtCod = CV_BGRA2RGB;
+					CvtCod = cv::COLOR_BGRA2RGB;
 					break;
 
 				default:
@@ -208,11 +208,11 @@ void ImageConvertNode::inputsUpdated( qint64 pTimeStamp )
 			switch( mDestinationFormat )
 			{
 				case fugio::ImageFormat::RGB8:
-					CvtCod = CV_YUV2RGB_Y422;
+					CvtCod = cv::COLOR_YUV2RGB_Y422;
 					break;
 
 				case fugio::ImageFormat::GRAY8:
-					CvtCod = CV_YUV2GRAY_Y422;
+					CvtCod = cv::COLOR_YUV2GRAY_Y422;
 					break;
 
 				default:
@@ -227,19 +227,19 @@ void ImageConvertNode::inputsUpdated( qint64 pTimeStamp )
 			switch( mDestinationFormat )
 			{
 				case fugio::ImageFormat::RGB8:
-					CvtCod = CV_GRAY2RGB;
+					CvtCod = cv::COLOR_GRAY2RGB;
 					break;
 
 				case fugio::ImageFormat::RGBA8:
-					CvtCod = CV_GRAY2RGBA;
+					CvtCod = cv::COLOR_GRAY2RGBA;
 					break;
 
 				case fugio::ImageFormat::BGR8:
-					CvtCod = CV_GRAY2BGR;
+					CvtCod = cv::COLOR_GRAY2BGR;
 					break;
 
 				case fugio::ImageFormat::BGRA8:
-					CvtCod = CV_GRAY2BGRA;
+					CvtCod = cv::COLOR_GRAY2BGRA;
 					break;
 
 				default:
unchanged:
--- a/plugins/OpenCV/inpaintnode.cpp
+++ b/plugins/OpenCV/inpaintnode.cpp
@@ -72,7 +72,7 @@ void InPaintNode::inputsUpdated( qint64 pTimeStamp )
 	cv::Mat		MatMsk = OpenCVPlugin::image2mat( MskImg );
 	cv::Mat		MatTmp;
 
-	cv::cvtColor( MatSrc, MatTmp, CV_BGRA2RGB );
+	cv::cvtColor( MatSrc, MatTmp, cv::COLOR_BGRA2RGB );
 
 	cv::inpaint( MatTmp, MatMsk, mMatImg, mPinInputRadius->value().toDouble(), cv::INPAINT_NS );
 
