diff -ru milkytracker-0.90.86/src/tracker/Makefile.am milkytracker-0.90.86-alsalib/src/tracker/Makefile.am
--- milkytracker-0.90.86/src/tracker/Makefile.am	2013-11-09 12:57:21.000000000 +0200
+++ milkytracker-0.90.86-alsalib/src/tracker/Makefile.am	2016-01-24 21:35:50.811383214 +0200
@@ -76,7 +76,7 @@
 $(top_builddir)/src/ppui/osinterface/libosinterface.a \
 $(top_builddir)/src/fx/libfx.a $(compressionLib) $(RTMIDI) $(SDL_LIBS) \
 $(top_builddir)/src/compression/zziplib/generic/libzzip.a \
-$(Z_LIBS)
+$(Z_LIBS) $(ALSA_LIBS)
 #$(ZZIP_LIBS) read below
 
 # NOTE: The MilkyTracker source contains a MODIFIED version of the zzip
