diff -ru milkytracker-0.90.86/src/midi/Makefile.am milkytracker-0.90.86-system-rtmidi/src/midi/Makefile.am
--- milkytracker-0.90.86/src/midi/Makefile.am	2013-11-09 12:57:21.000000000 +0200
+++ milkytracker-0.90.86-system-rtmidi/src/midi/Makefile.am	2016-01-24 22:27:45.054750663 +0200
@@ -1,8 +1,8 @@
 noinst_LIBRARIES = libmidi.a
 
-libmidi_a_SOURCES = posix/MidiReceiver_pthread.cpp rtmidi/RtMidi.cpp
+libmidi_a_SOURCES = posix/MidiReceiver_pthread.cpp
 
-noinst_HEADERS = posix/MidiReceiver_pthread.h rtmidi/RtError.h rtmidi/RtMidi.h \
+noinst_HEADERS = posix/MidiReceiver_pthread.h \
 MidiTools.h
 
 AM_CPPFLAGS=-D__LINUX_ALSASEQ__ -D__LINUX__ -DMILKYTRACKER ${ALSA_CFLAGS} \
diff -ru milkytracker-0.90.86/src/midi/posix/MidiReceiver_pthread.cpp milkytracker-0.90.86-system-rtmidi/src/midi/posix/MidiReceiver_pthread.cpp
--- milkytracker-0.90.86/src/midi/posix/MidiReceiver_pthread.cpp	2013-11-09 02:11:58.000000000 +0200
+++ milkytracker-0.90.86-system-rtmidi/src/midi/posix/MidiReceiver_pthread.cpp	2016-01-30 21:44:35.243989528 +0200
@@ -32,7 +32,6 @@
 #include "Tracker.h"
 #include "PPMutex.h"
 #include "RtMidi.h"
-#include "RtError.h"
 #include "PPSystem.h"
 #include <iostream>
 #include "MidiTools.h"
@@ -64,7 +63,7 @@
 		midiin = new RtMidiIn();
 		midiin->openPort(deviceID);
 	}
-	catch (RtError &error) 
+	catch (RtMidiError &error) 
 	{
 		error.printMessage();
 		goto cleanup;
diff -ru milkytracker-0.90.86/src/tracker/Makefile.am milkytracker-0.90.86-system-rtmidi/src/tracker/Makefile.am
--- milkytracker-0.90.86/src/tracker/Makefile.am	2013-11-09 12:57:21.000000000 +0200
+++ milkytracker-0.90.86-system-rtmidi/src/tracker/Makefile.am	2016-01-30 22:19:46.557714000 +0200
@@ -76,7 +76,7 @@
 $(top_builddir)/src/ppui/osinterface/libosinterface.a \
 $(top_builddir)/src/fx/libfx.a $(compressionLib) $(RTMIDI) $(SDL_LIBS) \
 $(top_builddir)/src/compression/zziplib/generic/libzzip.a \
-$(Z_LIBS) $(ALSA_LIBS)
+$(Z_LIBS) $(ALSA_LIBS) -lrtmidi
 #$(ZZIP_LIBS) read below
 
 # NOTE: The MilkyTracker source contains a MODIFIED version of the zzip
