diff -ru milkytracker-0.90.86/src/compression/zziplib/generic/fseeko.cpp milkytracker-0.90.86-zziplib-fixes/src/compression/zziplib/generic/fseeko.cpp
--- milkytracker-0.90.86/src/compression/zziplib/generic/fseeko.cpp	2013-11-09 02:11:58.000000000 +0200
+++ milkytracker-0.90.86-zziplib-fixes/src/compression/zziplib/generic/fseeko.cpp	2016-01-30 21:52:24.503454707 +0200
@@ -144,7 +144,7 @@
 	name[len] = '\0';
 	return name;
     }
-    ___ auto struct zzip_file_header header;
+    ___ struct zzip_file_header header;
     if (zzip_entry_fread_file_header (entry, &header) 
 	&& ( len = zzip_file_header_namlen(&header) )) {
 	char* name = (char*)malloc (len+1);
